client/src/containers/EditUserProfile/styles.css

```diff
@@ -63,6 +65,8 @@
     flex-direction: row;
```

Comment:
This (and line below) will works if you set `display: flex`

Reply:
Yes, exactly, it is written just the line before... 😉 😄